public/app/features/templating/template_srv.ts

```diff
@@ -154,6 +154,19 @@ export class TemplateSrv implements BaseTemplateSrv {
       return format(value, variable, this.formatValue);
     }
 
+    if (!format) {
+      format = 'glob';
```

Comment:
Could a default parameter be used instead? Like
```typescript
formatValue(value: any, format: any = 'glob', variable: any) {
```

Reply:
don't think you can use a default value for the non-optional argument (difference between ? and being undefined)